### PR TITLE
fix: tslint the generated package properly

### LINKED ISSUE #64: TSlint in generated result is not finding the errors.

### PR DESCRIPTION
**Pull request by BelfordZ**
fixes #64

### CODE REVIEW

File: templates/js/templated/exported-class.template.js [hunk 1]
```diff
@@ -48,10 +47,10 @@ export default class <%= className %> {
           "Expected param in position ",
           index,
           " to match the json schema: ",
-          JSON.stringify(param.schema, undefined, '\t'),
+          JSON.stringify(param.schema, undefined, "\t"),
```
Comment (BelfordZ) [inline templates/js/templated/exported-class.template.js]:
```suggestion
          JSON.stringify(param.schema, undefined, "  "),
```

File: templates/js/templated/exported-class.template.js [hunk 2]
```diff
@@ -5,38 +5,37 @@ import _ from "lodash";
...
-  rpc: jayson.Client;
-  methods: Array<any>;
-  validator: ajv.Ajv;
+  public rpc: jayson.Client;
+  public methods: any[];
+  private validator: ajv.Ajv;
 
   constructor(options: any) {
-    this.methods = <%= JSON.stringify(methods) %>;
+    this.methods = <%= JSON.stringify(methods, undefined, '  ') %>;
```
Comment (BelfordZ) [inline templates/js/templated/exported-class.template.js]:
```suggestion
    this.methods = <%= JSON.stringify(methods, undefined, "  ") %>;
```

File: src/index.test.js [hunk 3]
```diff
@@ -21,6 +21,8 @@ describe(`Examples to generate Js clients`, () => {
     return await rmdir(testDir);
   });
 
+  // const example = examples.petstoreExpanded; // uncomment to run specific example
```
Comment (shanejonas) [inline src/index.test.js]: how does this work if i i comment it? the next line loops

Comment (shanejonas) [reply src/index.test.js]: just delete it lol

Comment (BelfordZ) [inline src/index.test.js]:
```suggestion
```

**Pull request merged**